### PR TITLE
Updated the contribution.md

### PR DESCRIPTION
**Pull request by palashg7563**
Issue refered to #34 
@abhishek71994 please review as changelog and issue log file are created please free to comment if anything is missing or wrongly placed

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# CODE OF CONDUCT
+
+This code of conduct outlines our expectations for participants within the Facebook Open Source community, as well as steps to reporting unacceptable behavior. We are committed to providing a welcoming and inspiring community for all and expect our code of conduct to be honored. Anyone who violates this code of conduct may be banned from the community.
```
Comment (Yugandhartripathi) [inline CODE_OF_CONDUCT.md]: Facebook?

File: contributing.md [hunk 2]
```diff
@@ -1,25 +1,83 @@
...
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Pull Request Process
+
+1. Create a fork
+1. Create your feature branch: `git checkout -b {user}/{feature}`
+1. Inform the channel which feature are you working on
+1. Commit your changes: `git commit -am 'Add some feature'`
+1. Once the work is done, pull any new changes from the “staging” branch and resolve any conflicts.
```
Comment (abhishek71994) [inline contributing.md]: This is a great input, would definitely help resolve conflicts as the maintainer would not be having trivial headaches.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (palashg7563) [reply CODE_OF_CONDUCT.md]: oops 😅

File: contributing.md (hunk 2 above)
Comment (palashg7563) [reply contributing.md]: Yes 😃

**Comment by palashg7563** [comment]
Please merge if there is not conflict @abhishek71994 @Yugandhartripathi

File: contributing.md [hunk 3]
```diff
@@ -1,25 +1,83 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+### Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at [INSERT EMAIL ADDRESS]. All
```
Comment (abhishek71994) [inline contributing.md]: Please insert abhishek71994@gmail.com for the moment.

Comment (palashg7563) [reply contributing.md]: Ohk

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# CODE OF CONDUCT
+
+This code of conduct outlines our expectations for participants within the GirlScript Foundation [Link](https://www.gssoc.tech), as well as steps to reporting unacceptable behavior. We are committed to providing a welcoming and inspiring community for all and expect our code of conduct to be honored. Anyone who violates this code of conduct may be banned from the community.
```
Comment (abhishek71994) [inline CODE_OF_CONDUCT.md]: WrippleFOSS instead of girlscript

website link be https://wripplefoss.github.io/

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  * Violent threats or language directed against another person.
+  * Discriminatory jokes and language.
+  * Posting sexually explicit or violent material.
+  * Posting (or threatening to post) other people’s personally identifying information (“doxing”).
+  * Personal insults, especially those using racist or sexist terms.
+  * Unwelcome sexual attention.
+  * Advocating for, or encouraging, any of the above behavior.
+  * Repeated harassment of others. In general, if someone asks you to stop, then stop.
+  * When we disagree, try to understand why: Disagreements, both social and technical, happen all the time. It is important that we resolve disagreements and differing views constructively.
+  * Remember that we’re different. The strength of our community comes from its diversity, people from a wide range of backgrounds. Different people have different perspectives on issues. Being unable to understand why someone holds a viewpoint doesn’t mean that they’re wrong. Don’t forget that it is human to err and blaming each other doesn’t get us anywhere. Instead, focus on helping to resolve issues and learning from mistakes.
```
Comment (abhishek71994) [inline CODE_OF_CONDUCT.md]: Agreed 100%

File: CODE_OF_CONDUCT.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 # CODE OF CONDUCT
 
-This code of conduct outlines our expectations for participants within the GirlScript Foundation [Link](https://www.gssoc.tech), as well as steps to reporting unacceptable behavior. We are committed to providing a welcoming and inspiring community for all and expect our code of conduct to be honored. Anyone who violates this code of conduct may be banned from the community.
+This code of conduct outlines our expectations for participants within the WrippleFOSS [Link](https://wripplefoss.github.io/), as well as steps to reporting unacceptable behavior. We are committed to providing a welcoming and inspiring community for all and expect our code of conduct to be honored. Anyone who violates this code of conduct may be banned from the community.
```
Comment (Yugandhartripathi) [inline CODE_OF_CONDUCT.md]: Instead of displaying the word "Link" explicitly why not attach the link to WrippleFOSS. What do you think @abhishek71994 ?

**Pull request merged**